web/styles/search.css

```diff
@@ -13,8 +13,8 @@
         grid-area: search-icon;
         align-self: center;
         text-decoration: none;
-        padding: 0 7px;
-        font-size: 17px;
+        padding: 0 0.5em; /* 7px at 14px em */
```

Comment:
This calculation is wrong. Because the font-size is being set to the em equivalent of 17px, that's the em value needed here: `/* 7px at 17px/em */